docs/api/drivers/Watchdog.md

```diff
@@ -1,11 +1,49 @@
 # Watchdog
 
-[Add description here.]
+Use the Watchdog interface to set up a hardware watchdog. If you fail to refresh the watchdog periodically, it will reset the system after a set period of time.
```

Comment:
Why would you want to set up a hardware watchdog? What are the use cases?

Reply:
Updated.